client/client.go

```diff
@@ -31,7 +31,9 @@ type RemoteStore interface {
 	//
 	// `file` should be relative to the root of remote repository (e.g.
 	// "root.json" or "targets/path/to/target/file.txt")
-	Get(file string, size int64) (io.ReadCloser, error)
+	//
+	// ErrNotFound should be returned if the given file does not exist.
+	Get(file string) (io.ReadCloser, int64, error)
```

Comment:
This is good, though we should make sure that a sentinel value of `-1` for the size signals that the implementation does not know the length.